packages/react-email/source/commands/dev.ts

```diff
@@ -1,28 +1,20 @@
-import fs from 'node:fs';
+import { exists } from 'fs-extra';
 import shell from 'shelljs';
-import { downloadClient, REACT_EMAIL_ROOT } from '../utils';
+
 import { setupServer } from '../utils/run-server';
 
 interface Args {
   dir: string;
   port: string;
-  skipInstall: boolean;
 }
 
-export const dev = async ({ dir, port, skipInstall }: Args) => {
+export const dev = async ({ dir, port }: Args) => {
   try {
-    if (!fs.existsSync(dir)) {
+    if (!(await exists(dir))) {
```

Comment:
any reason to switch from `fs.existsSync` from the default node package to `fs-extra`?

Reply:
No real reason, just wanted to make the code use the same thing everywhere, but fs-extra also has a `existsSync` and it looks better than awaiting inline